pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -147,13 +147,24 @@ def replay_f2c(args: list[str], dryrun: bool = False) -> list[str] | None:
     new_args = ["gcc"]
     found_source = False
     for arg in args[1:]:
-        if arg.endswith(".f"):
-            filename = os.path.abspath(arg)
+        if arg.endswith(".f") or arg.endswith(".F"):
+            filepath = Path(arg).resolve()
             if not dryrun:
                 fix_f2c_input(arg)
-                subprocess.check_call(
-                    ["f2c", os.path.basename(filename)], cwd=os.path.dirname(filename)
-                )
+                if arg.endswith(".F"):
+                    subprocess.check_call(
```

Comment:
So the standard way of not actually running the command doesn't work for .f. Might be worth adding a comment.

Reply:
Yeah apparently `.F` commands expect first to be run through the C preprocessor (they have #ifdef's in them)